rooutil/inc/cutflow.hh

```diff
@@ -18,14 +18,23 @@ usage:
 
 #ifndef cutflow_hh_
 #define cutflow_hh_
+
+#include "TH1.h"
+
 #include <format>
 // Structure to hold cutflow data
 struct CutflowEntry {
+    /*
+    Flexible struct. Two field "cut_name" e.g t>600 ns and "passed_count" e.g. 1000 events
+    */
     std::string cut_name;
```

Comment:
No need to implement in this PR but could we use the cut functions here? Than the user doesn't need to pass in the passed_count and we can get this function to do the counting too...